.gitlab-ci.yml

```diff
@@ -400,24 +406,18 @@ build-rustdoc:
   after_script:
     - buildah logout --all
 
-publish-polkadot-image:
+# This image is used in testnets
+# Release image is handled by the Github Action here:
+# .github/workflows/publish-docker-release.yml
+publish-polkadot-debug-image:
   stage:                           build
   <<:                              *build-push-image
+  <<:                              *common-refs
   variables:
     <<:                            *image-variables
     # scripts/dockerfiles/polkadot_injected_debug.Dockerfile
     DOCKERFILE:                    dockerfiles/polkadot_injected_debug.Dockerfile
     IMAGE_NAME:                    docker.io/paritypr/synth-wave
```

Comment:
Would it be possible to rename the docker image to `paritypr/polkadot-debug` ?

Reply:
For me the naming is ok, need to ask @TriplEight and @pepoviola. 

@pepoviola do you have any dependencies in `zombienet` related to `synth-wave` image and its current name?

Reply:
as I mentioned in the issue

> [Synth](https://fallout.fandom.com/wiki/Synth)- stands not for the musical genre, but rather for synths, synthetic humanoids that would habitate the simnet. Any suggestion to a better name? `paritytech/polkadot` will be confusing, one can find themself on a host with a `polkadot` process running without realizing if the host is prod or not.

The name was a fit for `simnet`, but not it's no longer. I'm OK with renaming the image. Maybe something like `polkadot-goule` would keep us less formal?

Reply:
@TriplEight  Is `polkadot-debug` not a suitable name ? This image is not only needed for the Zombienet but also for Testnets (Westend/Rococo/...) or even potentially production networks for example to enable debug symbols for easy profiling.

Reply:
@PierreBesson  nah, just wanted to make it less boring. If you feel this would work better, let's rename it, of course.